test/unit/instrumentation/http2/http2.test.js

```diff
@@ -0,0 +1,684 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+const assert = require('node:assert')
+const test = require('node:test')
+const DESTINATIONS = require('../../../../lib/config/attribute-filter').DESTINATIONS
+const NAMES = require('../../../../lib/metrics/names')
+const hashes = require('../../../../lib/util/hashes')
+const helper = require('../../../lib/agent_helper')
+const Shim = require('../../../../lib/shim').Shim
+const createHttp2ResponseServer = require('./fixtures/http2')
+const events = require('node:events')
+
+test.beforeEach(async (ctx) => {
+  const { server, baseUrl, responses, host, port } = await createHttp2ResponseServer()
+  ctx.nr = {}
+  ctx.nr.initialize = require('../../../../lib/instrumentation/core/http2')
```

Comment:
when you require http2 and agent is loaded, it should call this instrumentation. there's no reason for us to manually call this instrumentation.